assets/src/scripts/monitoring-location/components/hydrograph/graph-brush.js

```diff
@@ -90,7 +126,7 @@ export const drawGraphBrush = function(container, store) {
                 selection = xScale.range();
             }
             if (selection[1] - selection[0] > 0) {
-                graphBrush.move(group, selection);
+                group.call(graphBrush.move, selection);
```

Comment:
This does the same thing, FWIW. I'm not sure there is a preference other than my own.

Reply:
Yup, no preference here either. I was just experimenting and forgot switch it back.